### PR TITLE
Add Dynamic Product and Look Page

### PR DESCRIPTION
**Pull request by ibrahimbutt**
Adds a dynamic product and look page, by keeping state in `App`.

### CODE REVIEW

File: src/LookPage/index.js [hunk 1]
```diff
@@ -75,57 +74,56 @@ const ButtonContainer = styled.div`
...
-      },
-    ],
-    currentLook: {
-      id: 1,
-      image: 'https://i.imgur.com/n1IqG2c.jpg',
-      brands: ['Ralph Lauren', 'Armani'],
-      description: 'The description',
-    },
-  }
+  // state = {
```
Comment (nkhil) [inline src/LookPage/index.js:73]: I think it's safe to delete the commented out code now since it also exists in App?

**Review by nkhil — APPROVED** [review]

File: src/LookPage/index.js (hunk 1 above)
Comment (kaihoffman) [reply src/LookPage/index.js:73]: This fails tests - is this intentional?

**Comment by kaihoffman** [comment]
OK, I've managed to get the Cypress errors sorted but there's still Jest tests that fail. There's a bunch of commented-out code in there @ibrahimbutt  - do you know why? It looks a bit like that's why these tests fail.

**Review by kaihoffman — APPROVED** [review]
Tests passing = Happy Kai

**Review by KrzysztofBalejko — APPROVED** [review]
Discussed with Kai.

**Pull request merged**